openmetadata-ui/src/main/resources/ui/src/constants/Widgets.constant.ts

```diff
@@ -59,16 +59,71 @@ export const WIDGETS_MORE_MENU_KEYS = {
 export const WIDGETS_MORE_MENU_OPTIONS = [
   {
     key: WIDGETS_MORE_MENU_KEYS.HALF_SIZE,
-    label: i18n.t('label.half-size'),
+    label: t('label.half-size'),
   },
   {
     key: WIDGETS_MORE_MENU_KEYS.FULL_SIZE,
-    label: i18n.t('label.full-size'),
+    label: t('label.full-size'),
   },
   {
     key: WIDGETS_MORE_MENU_KEYS.REMOVE_WIDGET,
-    label: i18n.t('label.remove-entity', {
-      entity: i18n.t('label.widget'),
+    label: t('label.remove-entity', {
+      entity: t('label.widget'),
     }),
   },
 ];
+
+export const FEED_WIDGET_FILTER_OPTIONS = [
+  {
+    label: t('label.all-activity'),
+    value: FeedFilter.ALL,
+    key: FeedFilter.ALL,
+  },
+  {
+    label: t('label.my-data'),
+    value: FeedFilter.OWNER,
+    key: FeedFilter.OWNER,
+  },
+  {
+    label: t('label.following'),
+    value: FeedFilter.FOLLOWS,
+    key: FeedFilter.FOLLOWS,
+  },
+];
+
+// Filter options for entity types
+export const FOLLOWING_WIDGET_FILTER_OPTIONS = [
+  {
+    label: t('label.latest'),
+    value: 'Latest',
+    key: 'Latest',
+  },
+  {
+    label: t('label.a-to-z'),
+    value: 'A to Z',
```

Comment:
Let's avoid using spaces for values and keys. They should `a-to-z` or `z-to-a`. There must already be an enum for this. If not, let's create and use it everywhere.